README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/brj/fish-cookbook) - Tips and recipes for fish-shell users
+- [fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup fish Shell with fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
> spending so many hours in the shell day by day and now in contact with one of their fathers

I think you should try the fish shell [gitter chat](https://gitter.im/fish-shell/fish-shell), the creators are there all the time answering questions, etc.